tests/unit/verticals/secret/output/test_json_output.py

```diff
@@ -495,3 +498,89 @@ def test_ignore_known_secrets(verbose, ignore_known_secrets, secrets_types):
         assert not incident_for_policy_break_type[policy_break.break_type][
             "incident_url"
         ]
+
+
+@pytest.mark.parametrize("verbose", [True, False])
```

Comment:
If I am not mistaken, the behavior does not depend on whether verbose is enabled or not, maybe we can remove this?